docs/.vuepress/config.js

```diff
@@ -42,7 +42,7 @@ module.exports = (_ctx) => ({
         lastUpdated: 'Last Updated',
         nav: require('./nav/en'),
         sidebar: {
-          '/guide/': getGuideSidebar('Guide', 'API Reference', 'Best Practices', 'Mobile', 'Resources'),
+          '/guide/': getGuideSidebar('Guide', 'API Reference', 'Best Practices', 'Mobile', 'Snaps', 'Resources'),
```

Comment:
I think we should wait to add snaps. I don't think much content/documentation on it yet. @rekmarks can clarify! If it's a no go, I'd be more than happy to work with you on generating documentation in the future for it if you're interested?

Reply:
Agreed - right now this is just an unused variable.